gpkit/model.py

```diff
@@ -544,6 +544,30 @@ def _latex(self, unused=None):
                                  for var, val in self.constants.items()]) +
                          ["\\end{array}"])
 
+    def interact(self,  fn_of_sol=None, ranges=None, **solveargs):
+        try:
+            from ipywidgets import interactive, FloatSlider
+        except ImportError:
+            from IPython.html.widgets import interactive, FloatSliderWidget
+            FloatSlider = FloatSliderWidget
+
+        if ranges is None:
+            ranges = {k._cmpstr: FloatSlider(min=v/10.0, max=10*v,
+                                             step=v/10.0, value=v)
```

Comment:
Hmm! I like that better than my initial thought of "half an order of magnitude ~ pi"